src/PrestaShopBundle/Form/Admin/Sell/Supplier/SupplierType.php

```diff
@@ -48,6 +48,15 @@
  */
 class SupplierType extends AbstractType
 {
+    const NAME_LENGTH = 64;
+    const PHONE_LENGTH = 32;
+    const ADDRESS_LENGTH = 128;
+    const POST_CODE_LENGTH = 12;
+    const CITY_NAME_LENGTH = 64;
+    const META_TITLE_LENGTH = 255;
+    const META_DESCRIPTION_LENGTH = 512;
+    const META_KEYWORD_LENGTH = 255;
```

Comment:
All these consts should be prefixed with `MAX_` as these are not required, but maximum lengths and maybe moved to `Domain`, @matks wdyt?

Reply:
👍